tests/forms_tests/field_tests/test_datetimefield.py

```diff
@@ -41,11 +41,8 @@ def test_datetimefield_clean(self):
             ('2014-09-23T22:34:41', datetime(2014, 9, 23, 22, 34, 41)),
             ('2014-09-23T22:34', datetime(2014, 9, 23, 22, 34)),
             ('2014-09-23', datetime(2014, 9, 23, 0, 0)),
-            ('2014-09-23T22:34Z', datetime(2014, 9, 23, 22, 34, tzinfo=utc)),
-            (
-                '2014-09-23T22:34+07:00',
-                datetime(2014, 9, 23, 22, 34, tzinfo=get_fixed_timezone(420)),
-            ),
+            ('2014-09-23T22:34Z', datetime(2014, 9, 23, 17, 34)),
+            ('2014-09-23T22:34+07:00', datetime(2014, 9, 23, 10, 34)),
```

Comment:
Not sure about these changes. the ticket is about the datetime object `timezone` and these now don't have timezone